v2/go.mod

```diff
@@ -80,3 +80,5 @@ require (
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 )
+
+replace github.com/leaanthony/winc v0.0.0-20220112092814-1608a1744133 => github.com/stffabi/winc v0.0.0-20220114143400-0c2341f6e43a
```

Comment:
Will be removed as soon as leaanthony/winc#9 has been merged

Reply:
Merged 🙏

Reply:
Thanks @leaanthony I've updated this PR.

Reply:
I've just merged a different change to winc. Could you possibly update go.mod to use the very latest? Thanks 🙏

Reply:
Both you and @taigrr are updating winc 😀

Reply:
Yes of course, I've just seen that :smile, give me just a couple of minutes.

Reply:
Done.

I'll come up with another PR for winc maybe in the next hour, but I think we should move this one forward and already merge this PR.